Dockerfile.openresty

```diff
@@ -40,6 +40,49 @@ RUN if [ "$EDITION" = "enterprise" ] ; then curl -fsSLo /tmp/nettle-${KONG_NETTL
     --with-lib-path="/tmp/gmp-${KONG_GMP_VERSION}/.libs/" \
     && make -j${RESTY_J}; fi
 
+ENV GOLANG_VERSION 1.13.5
+
+RUN set -eux; \
+	\
+# this "case" statement is generated via "update.sh"
```

Comment:
Minor: stale comment?

Reply:
I shamelessly stole this entire script. I'll remove the comment :)